packages/grafana-ui/src/components/Forms/Input/Input.tsx

```diff
@@ -246,7 +245,7 @@ export const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
 
         {(suffix || loading) && (
           <div className={styles.suffix} ref={suffixRef}>
-            {loading && <Icon name="spinner" className={cx('fa-spin', styles.loadingIndicator)} />}
+            {loading && <i className={cx('fa fa-spinner fa-spin', styles.loadingIndicator)} />}
```

Comment:
We currently don't have spinner icon, therefore we have temporarily migrated it to <i> + font awesome.

Reply:
Can we just hide it behind the Icon component too? Like having an IconType FontAwesome?